code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from "react";
+import { useParams, Link, Redirect } from "react-router-dom";
+
+import "../css/movieDetails.css";
+
+export const MovieDetails = () => {
+  const { movieId } = useParams();
+  const [movie, setMovie] = useState([]);
+
+  useEffect(() => {
+    fetch(
+      `https://api.themoviedb.org/3/movie/${movieId}?api_key=16cd57d89911f5854d96fcf791abf8a9&language=en-US`
+    )
+      .then((res) => res.json())
+      .then((json) => {
+        setMovie(json);
+      });
+  }, [movieId]);
+
+  if (movie.status_code === 34) {
+    return <Redirect to="/404" />;
+  } else {
+    return (
+      <section
+        className="details-page"
+        style={{
+          backgroundImage: `url(https://image.tmdb.org/t/p/w1280${movie.backdrop_path})`,
+        }}
+      >
+        <Link to="/" className="back-button">
+          <span className="fas fa-chevron-circle-left"></span>
```

Comment:
Is it possible to have another name so I understand what fas fa-chevron is? 🤔

Reply:
I don't think so since this is imported from fontawsome. Bur I can add a comment above to clearify things :)